<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LocationInfo</class>
 <widget class="QWidget" name="LocationInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>283</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form2</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>11</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="5" column="1">
    <widget class="QLineEdit" name="edtZip"/>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="edtState"/>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="edtCountry"/>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="edtCity"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="TextLabel12">
     <property name="text">
      <string>City:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="TextLabel14">
     <property name="text">
      <string>Zip code:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="TextLabel15">
     <property name="text">
      <string>Country:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="TextLabel13">
     <property name="text">
      <string>State:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="TextLabel11">
     <property name="text">
      <string>Address:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <spacer name="Spacer2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPlainTextEdit" name="edtStreet"/>
   </item>
   <item row="2" column="1">
    <widget class="QPlainTextEdit" name="edtExt"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
